calculation.js

```
@@ -0,0 +1,15 @@
+// Basic Mathematical Operations
+
+function basicOp(operation, value1, value2)
+{
+  let ans;
+  switch(operation){
+      case '+': ans = value1 + value2; break;
+      case '-': ans = value1 - value2; break;
+      case '*': ans = value1 * value2; break;
+      case '/': ans = value1 / value2;
```

Please also add a default case where you can throw errors like Invalid operations.